projects/kaltura-ng/kaltura-ui/src/lib/input-helper/input-helper.component.html

```diff
@@ -1,11 +1,18 @@
 <div class="kInputHelper" #inputHelperContainer>
-    <a class="kIconhelpLink" [class.kActive]="helperPopup.isShow" #helperLink><i class="kIconhelp_full"></i></a>
+    <div #helperLink>
+      <div #ref>
+        <ng-content select="[input-helper-content]"></ng-content>
+      </div>
+      <a *ngIf="ref.childNodes.length == 0" class="kIconhelpLink" [class.kActive]="helperPopup.isShow">
```

Comment:
lets make this `ref.childNodes.length === 0`

Reply:
sure, it's a typo